database/dbGenerator.js

```diff
@@ -1,206 +1,133 @@
-//     currId.photosAndComments.push(roomData);
-//     // Reset object to be empty
-//     roomData = {};
-//     // Add one to the counter
-//     count += 1;
-//   }
-// };
+writeManyHouses(houseWriter, 'utf8', () => console.log('Wrote house info.'));
+writeManyPhotos(photoWriter, 'utf8', () => console.log('Wrote photo info.'));
 
-// // Function that generates all rooms data
-// const dbGenerator = () => {
-//   // Create array to store the generated objects
-//   const db = [];
-//   // Iterate from 1 to 100 which will be the id number
-//   for (let i = 1; i <= 100; i += 1) {
-//     // Create empty object for current room id
-//     const currId = {};
-//     // Assign key 'id' and the iteration number as value
-//     currId.id = i;
-//     // Assign key 'photosAndComments' with an empty array
-//     currId.photosAndComments = [];
-//     // BEDROOM
-//     // Randomly generate number between 1 - 2
-//     roomDataGenerator(1, 2, imageUrls.urlPrefix, imageUrls.bedroomUrls, currId);
-//     // LIVING ROOM
-//     // If randomly generated number between 0 - 2 is greater than 0
-//     roomDataGenerator(0, 2, imageUrls.urlPrefix, imageUrls.livingRoomUrls, currId);
-//     // COUNTRY
-//     // If id is between 1 to 25
-//     if (currId.id >= 1 && currId.id <= 25) {
-//       // If randomly generated number between 0 - 4 is greater than 0
-//       roomDataGenerator(0, 4, imageUrls.urlPrefix, imageUrls.japanUrls, currId);
-//     }
-//     // If id is between 26 to 50
-//     if (currId.id >= 26 && currId.id <= 50) {
-//       // If randomly generated number between 0 - 4 is greater than 0
-//       roomDataGenerator(0, 4, imageUrls.urlPrefix, imageUrls.italyUrls, currId);
-//     }
-//     // If id is between 51 to 75
-//     if (currId.id >= 51 && currId.id <= 75) {
-//       // If randomly generated number between 0 - 4 is greater than 0
-//       roomDataGenerator(0, 4, imageUrls.urlPrefix, imageUrls.usUrls, currId);
-//     }
-//     // If id is between 76 to 100
-//     if (currId.id >= 76 && currId.id <= 100) {
-//       // If randomly generated number between 0 - 4 is greater than 0
-//       roomDataGenerator(0, 4, imageUrls.urlPrefix, imageUrls.singaporeUrls, currId);
-//     }
-//     // KITCHEN
-//     // If randomly generated number betweem 0 - 1 is greater than 0
-//     roomDataGenerator(0, 1, imageUrls.urlPrefix, imageUrls.kitchenUrls, currId);
-//     // BATHROOM
-//     // If randomly generated number between 0 - 1 is greater than 0
-//     roomDataGenerator(0, 1, imageUrls.urlPrefix, imageUrls.bathroomUrls, currId);
-//     // OTHERS
-//     // If randomly generated number between 0 - 4 is greater than 0
-//     roomDataGenerator(0, 4, imageUrls.urlPrefix, imageUrls.otherUrls, currId);
-//     // Push current room id data to db
-//     db.push(currId);
-//   }
-//   // Return the array
-//   return db;
-// };
+module.exports = {
+  dbGenerator,
+  // photoWasSelected,
```

Comment:
Should the comment here and the ones above be removed from the file? @brian-kim

Reply:
Yeah, it should have been removed. I'll push a new commit.